Roadmap.md

```diff
@@ -33,7 +41,49 @@ Tasks are tracked on a weekly basis through a dedicated spreadsheet.
 For more information see [Confidential Containers V0 Plan](https://docs.google.com/spreadsheets/d/1M_MijAutym4hMg8KtIye1jIDAUMUWsFCri9nq4dqGvA/edit#gid=0&fvid=1397558749).
 
 
-# Mid/long term roadmap 
+# Mid term roadmap
+
+Continue our journey using knowledge and support of Subject Matter Experts (SME's) in other 
+projects to form stronger 
+opinions on what is needed from components which can be integrated to deliver the confidential 
+containers objectives.
+
+- Harden the code used for the demos, 
+- improve CI/CD pipeline, 
```

Comment:
Nitty nit: Start with upper case?

Reply:
> 2 nits, and also: Do we really need the Sept and Oct intermediate milestones? Should we only keep the end goal to make things less confusing?

I am open though having Sept and Oct is fair reflection on what we did and how we thought about it?
Moving forward though I think releases will help and be ideal to use rather than months.